test/Octokit.Webhooks.Test/Converter/JsonStringEnumMemberConverterWithFallbackTests.cs

```diff
@@ -1,65 +0,0 @@
-namespace Octokit.Webhooks.Test.Converter;
-
-using System;
-using System.Globalization;
-using System.Reflection;
-using System.Text.Json;
-using System.Text.Json.Serialization;
-using FluentAssertions;
-using Octokit.Webhooks.Models.PingEvent;
-using Xunit;
-
-public class JsonStringEnumMemberConverterWithFallbackTests
-{
-    private readonly JsonSerializerOptions options = new()
-    {
-        Converters =
-        {
-            new JsonStringEnumMemberConverterWithFallback(),
-        },
-    };
-
-    [Theory]
-    [InlineData(@"""App""", HookType.App)]
-    [InlineData(@"""Organization""", HookType.Organization)]
-    [InlineData(@"""Repository""", HookType.Repository)]
-    [InlineData(@"""Foo""", HookType.Unknown)]
-    public void CanConvertString(string input, HookType expected)
-    {
-        var actual = JsonSerializer.Deserialize<HookType>(input, this.options);
-        actual.Should().Be(expected);
-    }
-
-    [Fact]
-    public void AllEnumerationsUseJsonStringEnumMemberConverterForJsonConverter()
-    {
-        var assembly = typeof(WebhookEvent).Assembly;
-        var types = assembly.GetTypes().ThatDeriveFrom<Enum>();
-
-        foreach (var type in types)
-        {
-            type.Should().BeDecoratedWith<JsonConverterAttribute>($"the {type.Name} enumeration must be decorated with [JsonConverter(...)]");
-
-            var converter = type.GetCustomAttribute<JsonConverterAttribute>();
-            converter.Should().NotBeNull();
-            converter!.ConverterType.Should().Be(typeof(JsonStringEnumMemberConverterWithFallback));
-        }
-    }
-
-    [Fact]
-    public void AllEnumerationsHaveUnknownMemberWithValueOfMinusOne()
```

Comment:
Maybe keep a version of this test somewhere that validates that all model properties use `StringEnum<TEnum>` instead of `TEnum` so new additions don't forget to use it?

Reply:
Added a test to ensure that there are no properties that derive directly from `System.Enum`